includes/reports/reports-functions.php

```diff
@@ -1210,7 +1210,7 @@ function filter_items( $report = false ) {
 	$report_id = $report->get_id();
 
 	// Bail if no report
-	if ( empty( $report_id ) ) {
+	if ( empty( $report_id ) || $report->has_errors() ) {
```

Comment:
I'm kind of inclined to remove this change in light of the new issue we've created: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/8468 That feels like the better fix.